src/providers/user/user-provider.js

```diff
@@ -92,6 +95,29 @@ export default class UserProvider extends EventEmitter {
         });
     }
 
+    async canSetMissionStatus() {
+        const user = await this.getCurrentUser();
+        const writeParameters = user.getWriteParameters();
+
+        const areParametersStatus = await Promise.all(
+            writeParameters.map(parameterName => isMissionStatusParameter(parameterName))
+        );
+
+        return areParametersStatus.some(isParameterStatus => isParameterStatus);
+    }
+
+    async getPossibleMissionActions() {
+        const possibleRoles = await this.missionStatus.getAllMissionActions();
```

Comment:
Change this parameter to `possibleActions`

Reply:
good catch!